receiver/netflowreceiver/README.md

```diff
@@ -0,0 +1,93 @@
+| [Code Owners](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/CONTRIBUTING.md#becoming-a-code-owner)    | [@evan-bradley](https://www.github.com/evan-bradley), [@dlopes7](https://www.github.com/dlopes7) |
+
+[development]: https://github.com/open-telemetry/opentelemetry-collector#development
+<!-- end autogenerated section -->
+
+The netflow receiver can listen for [netflow](https://en.wikipedia.org/wiki/NetFlow), [sflow](https://en.wikipedia.org/wiki/SFlow), and [ipfix](https://en.wikipedia.org/wiki/IP_Flow_Information_Export) data and convert it to OpenTelemetry logs. The receiver is based on the [goflow2](https://github.com/netsampler/goflow2) project.
+
+This gives OpenTelemetry users the capability of monitoring network traffic, and answer questions like:
+
+* Which protocols are passing through the network?
+* Which servers and clients are producing the highest amount of traffic?
+* What ports are involved in these network calls?
+* How many bytes and packets are being sent and received?
+
+## Getting started
+
+By default the receiver will listen for ipfix and netflow on port `2055` and sflow on port `6343`. The receiver can be configured to listen on different ports and protocols.
+
+Example configuration:
+
+```yaml
+receivers:
+  netflow:
+    - scheme: netflow
+      port: 2055
+      sockets: 16
+      workers: 32
+
+processors:
+  batch:
+    send_batch_size: 2000
+    timeout: 30s
+
+exporters:
+  debug:
+    verbosity: detailed
+
+service:
+  pipelines:
+    logs:
+      receivers: [netflow]
+      processors: [batch]
+      exporters: [debug]
+  telemetry:
+    logs:
+      level: debug
+```
+
+We recommend using the batch processor to reduce the number of log requests being sent to the exporter. The batch processor will batch log records together and send them in a single request to the exporter.
+
+You would then configure your network devices to send netflow, sflow, or ipfix data to the Collector on the specified ports.
+
+## Configuration
+
+| Field | Description | Examples | Default |
+|-------|-------------|--------| ------- |
+| scheme | The type of flow data that to receive | `sflow`, `netflow`, `flow` | `netflow` |
+| hostname | The hostname or IP address to bind to | `localhost` | `0.0.0.0` |
+| port | The port to bind to | `2055` | `2055` |
+| sockets | The number of sockets to use | 1 | 1 |
+| workers | The number of workers used to decode incoming flow messages | 2 | 2 |
```

Comment:
The underlying library suggests that the workers should be the # of CPU cores available and that there should be 2 workers for each socket. Do we want to mention those recommendations here under a performance header?

Reply:
@dlopes7 Do we still want to put in a performance header to describe these recommendations?